acq4/devices/Laser/devTemplate.py

```diff
@@ -26,104 +25,42 @@ def _translate(context, text, disambig):
-        self.outputPowerLabel.setFont(font)
-        self.outputPowerLabel.setText(_fromUtf8(""))
-        self.outputPowerLabel.setObjectName(_fromUtf8("outputPowerLabel"))
-        self.gridLayout_4.addWidget(self.outputPowerLabel, 0, 1, 1, 1)
-        self.samplePowerLabel = QtGui.QLabel(self.powerGroup)
-        font = QtGui.QFont()
-        font.setPointSize(11)
-        font.setBold(False)
-        font.setWeight(50)
-        self.samplePowerLabel.setFont(font)
-        self.samplePowerLabel.setText(_fromUtf8(""))
-        self.samplePowerLabel.setObjectName(_fromUtf8("samplePowerLabel"))
-        self.gridLayout_4.addWidget(self.samplePowerLabel, 1, 1, 1, 1)
-        self.label_8 = QtGui.QLabel(self.powerGroup)
-        self.label_8.setObjectName(_fromUtf8("label_8"))
-        self.gridLayout_4.addWidget(self.label_8, 1, 0, 1, 1)
-        self.powerAlertCheck = QtGui.QCheckBox(self.powerGroup)
-        self.powerAlertCheck.setChecked(True)
-        self.powerAlertCheck.setObjectName(_fromUtf8("powerAlertCheck"))
-        self.gridLayout_4.addWidget(self.powerAlertCheck, 1, 2, 1, 1)
-        self.gridLayout.addWidget(self.powerGroup, 0, 0, 1, 3)
-        self.wavelengthGroup = QtGui.QGroupBox(Form)
-        self.wavelengthGroup.setTitle(_fromUtf8(""))
-        self.wavelengthGroup.setObjectName(_fromUtf8("wavelengthGroup"))
-        self.gridLayout_5 = QtGui.QGridLayout(self.wavelengthGroup)
-        self.gridLayout_5.setMargin(6)
-        self.gridLayout_5.setObjectName(_fromUtf8("gridLayout_5"))
-        self.label_7 = QtGui.QLabel(self.wavelengthGroup)
-        self.label_7.setObjectName(_fromUtf8("label_7"))
-        self.gridLayout_5.addWidget(self.label_7, 0, 0, 1, 1)
-        self.wavelengthSpin = SpinBox(self.wavelengthGroup)
-        self.wavelengthSpin.setSuffix(_fromUtf8(""))
-        self.wavelengthSpin.setObjectName(_fromUtf8("wavelengthSpin"))
-        self.gridLayout_5.addWidget(self.wavelengthSpin, 0, 1, 1, 1)
-        self.wavelengthCombo = QtGui.QComboBox(self.wavelengthGroup)
-        self.wavelengthCombo.setObjectName(_fromUtf8("wavelengthCombo"))
-        self.wavelengthCombo.addItem(_fromUtf8(""))
-        self.gridLayout_5.addWidget(self.wavelengthCombo, 0, 2, 1, 1)
-        self.GDDEnableCheck = QtGui.QCheckBox(self.wavelengthGroup)
-        self.GDDEnableCheck.setObjectName(_fromUtf8("GDDEnableCheck"))
-        self.gridLayout_5.addWidget(self.GDDEnableCheck, 1, 0, 1, 1)
-        self.GDDSpin = QtGui.QSpinBox(self.wavelengthGroup)
-        self.GDDSpin.setMaximum(32000)
-        self.GDDSpin.setObjectName(_fromUtf8("GDDSpin"))
-        self.gridLayout_5.addWidget(self.GDDSpin, 1, 1, 1, 1)
-        self.GDDLimits = QtGui.QLabel(self.wavelengthGroup)
-        self.GDDLimits.setObjectName(_fromUtf8("GDDLimits"))
-        self.gridLayout_5.addWidget(self.GDDLimits, 1, 2, 1, 1)
-        self.gridLayout.addWidget(self.wavelengthGroup, 3, 0, 1, 3)
+        self.maitaiControlGroup = QtGui.QGroupBox(Form)
```

Comment:
something is not right here.. the base Laser gui should not contain any information about maitai. It probably doesn't need to be changed very much. The main differences I see are:

* Added maitai-specific status display
* Modified wavelength control without GDD

For the first, we should be able to find a way to add the status display widget via a subclass, rather than by modifying the base gui. For the second, we can have the base laser class just hide the GDD controls if the laser does not support it.